Gemfile

```diff
@@ -12,16 +12,20 @@ gem "stimulus-rails"
 gem "turbo-rails"
 gem "tzinfo-data", platforms: %i[ windows jruby ]
 
+group :development do
+  gem "web-console"
```

Comment:
purpose of the web console?

Reply:
The web-console gem provides an in-browser console for Rails applications in development, allowing developers to run Ruby code directly on web pages.